### PR TITLE
Add the option to define a port, username and password in the config.yaml

### PR DESCRIPTION
**Pull request by nielstron**
I just set the parsed settings from the config.yaml as object as `options` to [mqtt.connect](https://www.npmjs.com/package/mqtt#connect). By adding username, password and port those settings can be changed. [Accepted values](https://www.npmjs.com/package/mqtt#mqttclientstreambuilder-options) will apply.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -83,7 +83,8 @@ async.series([
     },
     function connectToMQTT (next) {
         winston.info('Connecting to MQTT at mqtt://%s', config.mqtt.host);
-        broker = mqtt.connect('mqtt://' + config.mqtt.host);
+		var mqtt_broker_options = config.mqtt;
```
Comment (stjohnjohnson) [inline server.js]: Looks like you're missing indentation here.

**Review by stjohnjohnson — CHANGES_REQUESTED** [review]
Looks very good.  Can you just fix the spacing please?

**Comment by nielstron** [comment]
So yeah the indentation should be fixed now

**Review by stjohnjohnson — APPROVED** [review]

**Comment by stjohnjohnson** [comment]
Thanks!

**Pull request merged**